loadgen/test_settings_internal.h

```diff
@@ -73,6 +73,10 @@ struct TestSettingsInternal {
   uint64_t schedule_rng_seed;
   uint64_t accuracy_log_rng_seed;
   double accuracy_log_probability;
+
+  bool issue_unique;
+  bool issue_same;
+  uint64_t performance_sample_count_override;
```

Comment:
One of TestSettingsInternal's responsibilities is select the final settings.
Can you remove "_override" from the name and assign it directly to the final value?
This will also help avoid the need to recalculate the final value in different places.

Reply:
My understanding was that PerformanceSampleCount will be derived from QSL unless the above test_setting is non-zero. So by directly assigning above to the final value, will it imply we will now be ignoring the value coming from QSL ?

Reply:
Not that we would be igorning the value coming from the QSL. I meant:

TestSettingsInternal::performance_sample_count = TestSettings::performace_sample_count_override == 0 ? QSL::PerformanceSampleCount() : TestSettings::performance_sample_count_override;